src/router.ts

```diff
@@ -289,7 +341,12 @@ export class Router {
   * @param config The route config.
   */
   createNavModel(config: RouteConfig): NavModel {
-    let navModel = new NavModel(this, 'href' in config ? config.href : config.route);
+    let navModel = new NavModel(
+      this,
+      'href' in config
+        ? config.href
+        // potential error when config.route is a string[] ?
```

Comment:
looks like it to me too

Reply:
Maybe this ?

https://github.com/aurelia/router/commit/0cd5423a421c17d6bf0b10598ea451902ec5a590#diff-15ee858ae5d5aa9b3e2846cfd1c4c466R303

But it still feels a bit awkward to me, maybe we need to improve this a bit